src/PJ_helmert.c

```diff
@@ -461,7 +461,7 @@ static PJ_OBS helmert_reverse_obs (PJ_OBS point, PJ *P) {
 #define ARCSEC_TO_RAD (DEG_TO_RAD / 3600.0)
 
 /***********************************************************************/
-PJ *PROJECTION(helmert) {
+PJ *OPERATION(helmert, PJ_IO_UNITS_METERS, PJ_IO_UNITS_METERS, 0) {
```

Comment:
Surely this is a `TRANSFORMATION`

Reply:
Yes (but also an operation, since operation is the union of transformations and conversions). Will, however, update, since there is no reason not using the most tight concept.